drivers/overlay/overlay.go

```diff
@@ -1539,11 +1539,6 @@ func (d *Driver) get(id string, disableShifting bool, options graphdriver.MountO
 	for err == nil {
 		absLowers = append(absLowers, filepath.Join(dir, nameWithSuffix("diff", diffN)))
 		diffN++
-		st, err = os.Stat(filepath.Join(dir, nameWithSuffix("diff", diffN)))
```

Comment:
Should line 1542 be kept? (Otherwise the for loop will not end)

Reply:
Doh! Yes, or rather, it should use `fileutils.Exists` now.

Reply:
ouch yes, fixed now